index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/css/main.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <title>CSS Framework</title>
+<body>
+    <div class="topnav"> 
+        <a href="#"> <img src="assets/images/logo.png" alt="" id="pexels"></a>
+        <div class="nav-right">
+            <a href="#">Explore</a>
+            <a href="#">License</a>
+            <a href="#">Join</a>
+            <a href="#"><i class="fas fa-ellipsis-h"></i></a>
+            <a href="#"><button class="button">Upload</button></a>
```

Comment:
Hey, this is the incorrect syntax, the button tag should not be wrapped in an anchor tag, either us an anchor tag or button, in this case, use anchor tag for consistency and style accordingly.